src/services/services.ts

```diff
@@ -591,7 +591,13 @@ namespace ts {
     }
 
     function findBaseOfDeclaration<T>(checker: TypeChecker, declaration: Declaration, cb: (symbol: Symbol) => T[] | undefined): T[] | undefined {
-        return firstDefined(declaration.parent ? getAllSuperTypeNodes(declaration.parent) : emptyArray, superTypeNode => {
+        const ClassOrInterfaceDeclaration = declaration.parent?.kind === SyntaxKind.Constructor ? declaration.parent.parent : declaration.parent;
+
+        if (!ClassOrInterfaceDeclaration) {
+            return;
+        }
+
+        return firstDefined(getAllSuperTypeNodes(ClassOrInterfaceDeclaration), superTypeNode => {

```

Comment:
Two formatting requests:
1. lowercase variables
2. No need for double-spacing in a small function.

```suggestion
        const classOrInterfaceDeclaration = declaration.parent?.kind === SyntaxKind.Constructor ? declaration.parent.parent : declaration.parent;
        if (!classOrInterfaceDeclaration) {
            return;
        }
        return firstDefined(getAllSuperTypeNodes(classOrInterfaceDeclaration), superTypeNode => {
```